mlir/lib/Transforms/Utils/DialectConversion.cpp

```diff
@@ -282,8 +282,10 @@ class CreateBlockRewrite : public BlockRewrite {
     block->dropAllUses();
```

Comment:
You can turn this one into `block->dropAllDefinedValueUses();` instead. `dropAllUses` is called from `dropAllDefinedValueUses`.

Reply:
done.